From ded4c82acedb373ed4393b69a9683c6bee38ff48 Mon Sep 17 00:00:00 2001
From: Alexander Tsoy <alexander@tsoy.me>
Date: Thu, 21 Nov 2013 20:33:38 +0400
Subject: [PATCH] usrmount: always install

/usr should be mounted by initramfs even if /sbin/init lives in /
---
 modules.d/98usrmount/module-setup.sh | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/modules.d/98usrmount/module-setup.sh b/modules.d/98usrmount/module-setup.sh
index d8a0ec2..31a6229 100755
--- a/modules.d/98usrmount/module-setup.sh
+++ b/modules.d/98usrmount/module-setup.sh
@@ -4,10 +4,7 @@
 
 # called by dracut
 check() {
-    local _init
     [[ $mount_needs ]] && return 1
-    _init=$(readlink -f /sbin/init)
-    [[ "$_init" == "${_init##/usr}" ]] && return 255
     return 0
 }
 
